app/screens/channel/channel.tsx

```diff
@@ -94,7 +93,7 @@ const Channel = ({
         return () => back?.remove();
     }, [componentId, isTablet]);
 
-    const marginTop = defaultHeight + (isTablet ? insets.top : 0);
+    const marginTop = defaultHeight;
```

Comment:
why this change?

Reply:
After checking the result of this change, I've corrected this code to reflect the following change. Because defaultHeader includes the `insets.top` measurement I needed subtract in from both values in the ternary operator.  I also verified the new messags toast appears again. It was misaligned.

<img width="521" alt="image" src="https://user-images.githubusercontent.com/7575921/196837079-ebe910b6-c941-4745-846a-a9973bf93d48.png">

With the change above. 
<img width="932" alt="image" src="https://user-images.githubusercontent.com/7575921/196837799-f734f742-c92c-411b-99fc-7cf859fd9a96.png">
<img width="446" alt="image" src="https://user-images.githubusercontent.com/7575921/196837860-4e96e6fa-29aa-4c20-80f0-e06e3270619f.png">